### DISCUSSION TITLE
[Flow] Adding "items.open" Event Hook

### DISCUSSION THREAD

**Discussion by halwesit** [directus/directus#18846] [Feature Requests]
### Summary

I have a case in my project, that we need to re-calculate items any time when we want to open it. I know that we can add a cron job to execute any task at any time we want, but I'm thinking of performance, the calculation is for a long list of users, but if we calculate any user at any time we want it would be better, also I see a manual trigger, now I use it, but I think it's not a user friendly and has a bad UX. 

### Detailed Design

![Screenshot 2023-06-08 154922](https://github.com/directus/directus/assets/10852176/fc21f010-d359-4b01-aef9-b18413d9ad2c)

**Comment by rijkvanzanten** [comment]
Please use the RFC format as per the form. This is missing some crucial details that make it hard to review and discuss 👍🏻
